app/controllers/crypto_coin.js

```diff
@@ -25,10 +27,29 @@ exports.listCoins = (req, res, next) => {
   const { id, currency } = req.user;
   return getCoinsByUser(id)
     .then(coins => convertCoins(coins, currency))
-    .then(coins => cryptoList(coins))
+    .then(coins => serializeCoins(coins))
     .then(coins => {
       logger.info('Finish list coins');
-      return res.send({ coins });
+      return res.send({ coins, currency });
+    })
+    .catch(next);
+};
+
+exports.topCoins = (req, res, next) => {
+  logger.info('Start list coins');
```

Comment:
Use a more specific message of what the controller is doing because this is the same as the previous one.